lib/fluent/plugin/in_tail.rb

```diff
@@ -593,13 +592,11 @@ def on_notify
           begin
             if @inode != inode || fsize < @fsize
               # rotated or truncated
+              io = FileWrapper.open(@path) rescue nil
```

Comment:
Could you tell me when this io object will be closed? Or, not necessary to close? Why?